sklearn/ensemble/tests/test_bagging.py

```diff
@@ -63,7 +63,7 @@ def test_classification():
                           "bootstrap_features": [True, False]})
 
     for base_estimator in [None,
-                           DummyClassifier(),
+                           DummyClassifier(strategy='stratified'),
```

Comment:
I think we should rather explicitly ignore the warnings in the test and tag for 0.24.

Setting the strategy looks like an intentional thing that has to do with the test, when in reality it's just about avoiding warnings